<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelAnalyzerNGGUI</class>
 <widget class="RollupWidget" name="ChannelAnalyzerNGGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>778</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Channel Analyzer NG</string>
  </property>
  <widget class="QWidget" name="settingsContainer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>631</width>
     <height>131</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="DeltaFreqPowLayout">
      <item>
       <layout class="QHBoxLayout" name="DeltaFrequencyLayout">
        <item>
         <widget class="QToolButton" name="deltaMinus">
          <property name="toolTip">
           <string>Frequency shift direction</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <selectedoff>:/plus.png</selectedoff>
            <selectedon>:/minus.png</selectedon>
           </iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ValueDial" name="deltaFrequency" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Monospace</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>SizeVerCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="toolTip">
           <string>Demod shift frequency from center in Hz</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="deltaUnits">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>26</red>
                <green>26</green>
                <blue>26</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="BrightText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>26</red>
                <green>26</green>
                <blue>26</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="BrightText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>118</red>
                <green>118</green>
                <blue>117</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="BrightText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="text">
           <string> Hz</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ChannelPowerLayout">
        <item>
         <widget class="QLabel" name="channelPower">
          <property name="toolTip">
           <string>Channel power</string>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>0.0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="channelPowerUnits">
          <property name="text">
           <string> dB</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="SpanLayout">
      <item>
       <widget class="QLabel" name="spanLabel">
        <property name="text">
         <string>Rate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="spanLog2">
        <property name="toolTip">
         <string>Channel sample rate</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
        <property name="sliderPosition">
         <number>3</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>true</bool>
        </property>
        <property name="invertedControls">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="spanText">
        <property name="text">
         <string>6.0k</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="BWLayout">
      <item>
       <widget class="QLabel" name="BWLabel">
        <property name="text">
         <string>Hi</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="BW">
        <property name="toolTip">
         <string>Lowpass filter cutoff frequency</string>
        </property>
        <property name="minimum">
         <number>-60</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="BWText">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>3.0k</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lowCutLabel">
        <property name="text">
         <string>Low</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="lowCut">
        <property name="toolTip">
         <string>Highpass filter cutoff frequency (SSB)</string>
        </property>
        <property name="minimum">
         <number>-60</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lowCutText">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>0.3k</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ssb">
        <property name="toolTip">
         <string>SSB/DSB togggle</string>
        </property>
        <property name="text">
         <string>SSB</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="spectrumContainer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>720</width>
     <height>284</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>716</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Channel Spectrum</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutSpectrum">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="GLSpectrum" name="glSpectrum" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>250</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="GLSpectrumGUI" name="spectrumGUI" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="scopeContainer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>720</width>
     <height>314</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>716</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Channel Scope</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutScope">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="GLScopeNG" name="glScope" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>280</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="GLScopeNGGUI" name="scopeGUI" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollupWidget</class>
   <extends>QWidget</extends>
   <header>gui/rollupwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ValueDial</class>
   <extends>QWidget</extends>
   <header>gui/valuedial.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLSpectrum</class>
   <extends>QWidget</extends>
   <header>gui/glspectrum.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLSpectrumGUI</class>
   <extends>QWidget</extends>
   <header>gui/glspectrumgui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLScopeNG</class>
   <extends>QWidget</extends>
   <header>gui/glscopeng.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLScopeNGGUI</class>
   <extends>QWidget</extends>
   <header>gui/glscopenggui.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../sdrbase/resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
